packages/hardhat-core/sample-projects/javascript-esm/ignition/modules/LockModule.js

```diff
@@ -0,0 +1,14 @@
+import { buildModule } from "@nomicfoundation/hardhat-ignition/modules";
+
+const ONE_GWEI = 1_000_000_000n;
+
+export default buildModule("LockModule", (m) => {
+  const unlockTime = m.getParameter("unlockTime");
```

Comment:
I changed this to remove the default `unlockTime`, as a module shouldn't have a changing parameter (it would break reconciliation).
I left out the parameters json file that should have been swapped in to make the sample executable. My intention is to add in a sibling `parameters.json` file with a known future date (I am open to suggestions of the future date).

Reply:
I have added back in a default for `unlockTime`. It is now set to a point in time (rather than recalculating based on now).
The date picked is arbitrary.